### DISCUSSION TITLE
Errors related to "clear"

### DISCUSSION THREAD

**Discussion by YaYaYaRong** [dialogic-godot/dialogic#2278] [Q&A] [answered]
I'm very sorry to trouble you. I found that when I used "clear" to clear my previous text,my next text disappeared before it fully appeared,and I could only hear the sound of typing.May I ask if this is a bug or a setup issue for me?

**Comment by Jowan-Spooner** [comment]
Can you share some more info (dialogic version, godot version, excerpt from your timeline, screenshot, style, etc.)

**Reply by YaYaYaRong** [reply, marked as answer]
I just find that maybe it's becuase it needs some time to clear...when I let it wait for a moment,it works well...anyway thanks a lot!!!
